larray_editor/arraywidget.py

```diff
@@ -662,7 +662,9 @@ def __init__(self, parent, data=None, readonly=False, bg_value=None, bg_gradient
         # without this, we can crash python :)
         del painter, pixmap
         # select default gradient
-        gradient_chooser.setCurrentText(bg_gradient)
+        # requires Qt5+
+        # gradient_chooser.setCurrentText(bg_gradient)
+        gradient_chooser.setCurrentIndex(gradient_chooser.findText(bg_gradient))
```

Comment:
Does it work for both Qt4 and Qt5 or only for Qt4?

Reply:
I works on both.